The code to get the cgroup mount point went back to the last '/' in
the mount location.  Unfortunately, that didn't work because it would
change "/cgroup" to "".  Return the whole string.

Index: libvirt-1.0.6/src/util/vircgroup.c
===================================================================
--- libvirt-1.0.6.orig/src/util/vircgroup.c
+++ libvirt-1.0.6/src/util/vircgroup.c
@@ -2560,8 +2558,7 @@ static char *virCgroupIdentifyRoot(virCg
             return NULL;
         }
 
-        ignore_value(VIR_STRNDUP_QUIET(ret, group->controllers[i].mountPoint,
-                                       tmp - group->controllers[i].mountPoint));
+        ignore_value(VIR_STRDUP_QUIET(ret, group->controllers[i].mountPoint));
         return ret;
     }
 
